### ISSUE TITLE
qrEv should still work despite multi-session processes not being recommended

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2502]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.30.13](https://github.com/open-wa/wa-automate-nodejs/compare/4.30.12...4.30.13) (2022-02-14)

- 🚑  attempt qrEv fix #2502 [`#2502`](https://github.com/open-wa/wa-automate-nodejs/issues/2502)
- Release 4.30.13 [`34f265b`](https://github.com/open-wa/wa-automate-nodejs/commit/34f265b26bfc89d8a4ddbaa5dee492231bf1d03f)
- 📝  fix html errors in readme [`7882057`](https://github.com/open-wa/wa-automate-nodejs/commit/78820570672964d9db8681b31aad64bdcffc69d9)
- 🐛  prevent `autoEmoji` crashing whole process #2501 [`10a3298`](https://github.com/open-wa/wa-automate-nodejs/commit/10a3298ba468c723757e1e9cf4c7e71f44966610)
- 🐛 bugfix: prevent page errors crashing process via error.text [`2275a67`](https://github.com/open-wa/wa-automate-nodejs/commit/2275a6711634b98ab17b7b09a2d0e6459abb6480)
- updated types-only package [`839cb1c`](https://github.com/open-wa/wa-automate-nodejs/commit/839cb1cc784af7d0c5cda7650e6126c0a8a77323)
- 🙈 add new docs folder to .gitignore until new docs completed [`778e207`](https://github.com/open-wa/wa-automate-nodejs/commit/778e20788948f1442edabc8e0a45d7a3943f91ac)
